src/app/components/code-block/index.html

```diff
@@ -0,0 +1,56 @@
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the code block component, you must set it up as a dependency in your SKY UX SPA. First,install the package as part of your SPA dependencies: <stache-code>npm install @blackbaud/skyux-lib-code-block --save</stache-code>. Then install its plugin: <stache-code>npm install @blackbaud/skyux-builder-plugin-code-block --save</stache-code>. Next, add <stache-code>SkyCodeBlockModule</stache-code> as both an import and export in your <stache-code>app-extras.module.ts</stache-code> file. Finally, add the code block plugin to your <stache-code>skyuxconfig.json</stache-code> file <stache-code>"plugins": ["@blackbaud/skyux-builder-plugin-code-block"]</stache-code>.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="code"
+      isOptional="true"
+    >
+      Passes a code sample into the component. This property binds data, so you can use brackets to evaluate the template expression: <stache-code>[code]</stache-code>. Without the brackets, Angular treats the value as a string and initializes the target property with the string value instead of evaluating the data value. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideCopyToClipboard"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the copy-to-clipboard button in the code-block header. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="languageType"
+      isOptional="true"
+      defaultValue="markup"
+    >
+      Indicates the language of the code sample. The language type highlights the syntax in your code sample and appears in the header. This property accepts <stache-code>string</stache-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="hideHeader"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to hide the header portion of the code block. If the <stache-code>languageType</stache-code> is not declared and <stache-code>hideCopyToClipboard</stache-code> is set to true, the header is hidden. This property accepts <stache-code>boolean</stache-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
```

Comment:
Have you played around with options to display the language types? We need to include that somewhere. I meant to play around with it but have not gotten around to it. Have you attempted to just copy the setup from the Stache site?

Reply:
I added the content and files for this, but I just can't seem to figure out how to get it to display correctly, or rather, not display in two very narrow columns. I think it has to do with the way the demo pages are set up and this section falling outside of the typical sections.

Reply:
I think the section that we got in place last week works, so I'm resolving this comment.